dask/blockwise.py

```diff
@@ -215,11 +266,10 @@ class Blockwise(Layer):
         dependencies.  The inner-most elements of io_deps correspond to the
         mapping between place-holder collection indices, e.g ``(1,)``,
         and any chunk/partition-specific arguments needed by the underlying
-        IO function. If ``io_deps[key]`` corresponds to a tuple, the first
-        two elements of that tuple must contain the dask module path and the
-        name for the desired ``BlockwiseIODeps``-based mapping, respectively.
-        The remaining tuple elements should be initialization arguments.
-        See ``make_blockwise_graph`` for usage.
+        IO function. We assume that these mappings are all represented as
+        ``BlockwiseDep``-based object.
```

Comment:
I think this docstring is kinda out of date; `io_deps` is now just a single level `Dict[Str, BlockwiseDep]` right? Also could probably update the type annotations for `output_indices` and `io_deps` to include `BlockwiseDep`.